vignettes/rticles.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "Get started"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Get started}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Accessing templates
+
+```{r child="../man/fragments/usage.md"} 
+```
+
+
+View the list of available journal names with `rticles::journals()`.
+
+## Figure numbering & cross-references
```

Comment:
This is documented in https://bookdown.org/yihui/bookdown/a-single-document.html 

should we point to that too ? Or did you leave it out on purpose ?

Reply:
I think some redundancy here is helpful, could definitely add the pointer to the docs.

Reply:
Oh yes I agree with the redundancy. I just wanted to add a pointer to the doc.